### PR TITLE
eBPF Plugins for NetManager

### PR DESCRIPTION
**Pull request by MrSarius**
Just a draft PR for feedback. 

Issue: #182 

Code of my Masters Thesis.

Going to rebase onto develop soon. 🤙🏼

### CODE REVIEW

File: node-net-manager/env/tableQueryRequests.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ Asks the MQTT client for a table query and parses the result
 */
 func tableQueryByJobName(jobname string, force_optional ...bool) ([]TableEntryCache.TableEntry, error) {
 
-	log.Println("[MQTT TABLE QUERY] sname:", jobname)
+	log.Println("[MQTT TABLE QUERY] Sname:", jobname)
```
Comment (MrSarius) [inline node-net-manager/env/tableQueryRequests.go:32]: seems like my IDE did some stuff here. Going to fix!

File: node-net-manager/ebpfManager/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ebpf/**/**.o
+ebpf/**/**bpfeb.go
+ebpf/**/**bpfel.go
+ebpfManager
```
Comment (MrSarius) [inline node-net-manager/ebpfManager/.gitignore:4]: Can be remove. Just for dev...

File: node-net-manager/env/ContainerNetDeployment.go [hunk 3]
```diff
@@ -71,7 +72,7 @@ func (h *ContainerDeyplomentHandler) DeployNetwork(pid int, sname string, instan
 		return nil, nil, err
 	}
 
-	// set ip to the container veth
+	// set ip to the container Veth
```
Comment (MrSarius) [inline node-net-manager/env/ContainerNetDeployment.go:75]: seems like my IDE did some stuff here. Going to fix!

File: node-net-manager/env/EnvironmentManager.go [hunk 4]
```diff
@@ -189,8 +190,8 @@ func (env *Environment) ConfigureDockerNetwork(containername string) (string, er
 	return "", errors.New("not yet implemented")
 }
 
-// create veth pair and connect one to the host bridge
-// returns: bridgeVeth name, free Veth name, Vether interface to the veth pair and eventually an error
+// create Veth pair and connect one to the host bridge
```
Comment (MrSarius) [inline node-net-manager/env/EnvironmentManager.go:193]: seems like my IDE did some stuff here. Going to fix!

File: node-net-manager/env/UnikernelNetDeployment.go [hunk 5]
```diff
@@ -40,7 +40,7 @@ func (h *UnikernelDeyplomentHandler) DeployNetwork(pid int, sname string, instan
 		_ = netlink.LinkDel(veth)
 	}
 
-	logger.DebugLogger().Println("Creating veth pair for unikernel deployment")
+	logger.DebugLogger().Println("Creating Veth pair for unikernel deployment")
```
Comment (MrSarius) [inline node-net-manager/env/UnikernelNetDeployment.go:43]: seems like my IDE did some stuff here. Going to fix!

**Review by giobart — APPROVED** [review]

**Pull request merged**